package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fs-default-project-config",
-  "version": "6.1.3",
```

Comment:
this is a breaking change, as it will cause builds to fail

Reply:
I've adjusted the version number accordingly